### ISSUE TITLE
CopyZips needed

### ISSUE THREAD

**Issue by andrew-glenn** [aws-quickstart/connect-integration-aspect-wfm#8]
Lambda functions reference the the aws-quickstart bucket directly. We should migrate to the CopyZips pattern

```
onnect-integration-aspect-wfm    master ❯ egrep -i 'S3Key' -B1 -R *                4245   22:26:29
templates/aspect-wfm-rta.template-        S3Bucket: aws-quickstart
templates/aspect-wfm-rta.template:        S3Key: connect-integration-aspect-wfm/functions/packages/real-time-adherence/AspectKinesisLambda.zip
--
templates/aspect-wfm-ap-kda.template-        S3Bucket: aws-quickstart
templates/aspect-wfm-ap-kda.template:        S3Key: connect-integration-aspect-wfm/functions/packages/agent-productivity/lambda.zip
```

**Comment by DBurkhead** [comment]
OK, I looked at https://aws.amazon.com/blogs/infrastructure-and-automation/deploying-aws-lambda-functions-using-aws-cloudformation-the-portable-way/ and made changes to the two template files. There's a pull request for the develop branch.

**Comment by DBurkhead** [comment]
@andrew-glenn  Are any other changes required?

**Comment by andrew-glenn** [comment]
Resolved in #9
